pcdsdevices/pseudopos.py

```diff
@@ -40,6 +43,41 @@ def __init__(self, prefix='', parent=None, **kwargs):
             prefix = f'{parent.prefix}:{attr_name}'
 
         super().__init__(prefix=prefix, parent=parent, **kwargs)
+        self._verbose_name = verbose_name
+
+    def format_status_info(self, status_info):
+        """
+        Override status info handler to render the virtual motor.
+
+        Display virtual motor status info in the ipython terminal.
+
+        Parameters
+        ----------
+        status_info: dict
+            Nested dictionary. Each level has keys name, kind, and is_device.
+            If is_device is True, subdevice dictionaries may follow. Otherwise,
+            the only other key in the dictionary will be value.
+
+        Returns
+        -------
+        status: str
+            Formatted string with all relevant status information.
+        """
+        units = get_status_value(status_info, 'notepad_setpoint', 'units')
+        dial = get_status_value(status_info, 'dial_position', 'value')
+        position = get_status_value(status_info, 'position')
+
+        low, high = self.limits
+        name = self.prefix
+        if self._verbose_name:
+            name = f'{self._verbose_name} {self.prefix}'
+
+        return f"""\
+Virtual Motor {name}
+Current position (user, dial): {position}, {dial} [{units}]
+User limits (low, high): {low}, {high} [{units}]
```

Comment:
Are the `units` the same for limits as for the current position? I assumed they are the same here

Reply:
Yes, the limits should always have the same units as the current position